source/installation/kafka-deployments/migrator-with-kafka.txt

```diff
@@ -0,0 +1,81 @@
+.. _kafka-intro:
+
+======================================
+Relational Migrator with Kafka Connect
```

Comment:
Can we change this heading, maybe to something like "Integrating Relational Migrator with Kafka"? Kafka Connect is just one component of the solution.

Reply:
Updated.